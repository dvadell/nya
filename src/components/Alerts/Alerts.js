import React from 'react'
import translations from './Alerts.translations'
import Localizer from '../../lib/Localizer'

const Localize = Localizer(translations)

const Alerts = () => {
    return (<div>
                <h1><Localize>Alerts</Localize></h1>
                <p><Localize>In a future version...</Localize></p>
            </div>)
} 

export default Alerts