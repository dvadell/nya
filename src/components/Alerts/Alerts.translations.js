const translation = {
    'Alerts': {
        es_ar:  'Alertas',
        es_es:  'Alertas',
        en:     'Alerts'
    },
    'In a future version...': {
        es_ar:  'En una versión futura, vas a poder recibir alertas de nuevo gatos encontrados dentro de un cierto radio. Pero me va a tomar tiempo.',
        es_es:  'En una versión futura, será posible recibir alertas de nuevos gatos encontrados dentro de un cierto radio. Pero me va a tomar tiempo.',
        en:     'In a future version, you\'ll be able to receive alerts of new cats found within a location. But it will take me some time.' 
    },
}

export default translation;