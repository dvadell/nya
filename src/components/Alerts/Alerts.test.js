import { shallow } from 'enzyme'
import React from 'react';
import Alerts from './Alerts';

describe('Alerts', () => {
    it('expect to render empty Alerts component', () => {
    expect(shallow(<Alerts />)).toMatchSnapshot()
    })
})

