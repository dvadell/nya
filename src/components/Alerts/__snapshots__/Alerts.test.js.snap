// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alerts expect to render empty Alerts component 1`] = `ShallowWrapper {}`;
